minio/minio_test.go

```diff
@@ -0,0 +1,172 @@
+package minio
+
+import (
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/gofiber/utils"
```

Comment:
Use testify.Require instead of utils.AssertEqual. 

See here: https://pkg.go.dev/github.com/stretchr/testify/require

Reply:
changed